### DISCUSSION TITLE
There is no script for my electricity meter, how can I create it myself?

### DISCUSSION THREAD

**Discussion by Amazonasmann** [arendst/Tasmota#22304] [Support request]
Hi everyone,

I have a Latronic L20-1Z1-20A0 electricity meter. Sadly there is no script available in https://tasmota.github.io/docs/Smart-Meter-Interface/ for which I want to read with a Hichi IR reader. I dont find anything in google regarding that specific device. How can I create a script for that device, can someone give me a hint on how to start?

**Comment by Amazonasmann** [comment]
bump

**Reply by sfromis** [reply]
https://tasmota.github.io/docs/Smart-Meter-Interface/#descriptor-syntax

**Reply by 001Tycoon** [reply]
Hello
Tasmota command sensor53 d1 when you receive data you can decode it on the website to create a script.
Tasmota SML Dekoder
https://tasmota-sml-parser.dicp.net/
